src/tests/ut/medicine/medicine_test.cpp

```diff
@@ -4,14 +4,50 @@
 
 struct MedicineTest : ::testing::Test
 {
-    std::shared_ptr<Product> anaesthetic = std::make_shared<Medicine>(
-        std::string{"anaesthetic"}, 45, 100, 10122025, std::pair{-10, 0}, std::vector<std::string>{{"chemicals"}});
+    std::chrono::year_month_day expirationDate{static_cast<std::chrono::year>(2030), std::chrono::January,
+                                               static_cast<std::chrono::day>(31)};
```

Comment:
perhaps isnteaad of static casting, easier would to be to use constant literal suffices, like this (copied from cppreference):
```
constexpr auto ymd2 = std::chrono::year_month_day(
        2020y, std::chrono::January, 31d // overload (2)
    );
```

Reply:
![obraz](https://github.com/coders-school/desktop-business-app/assets/94867075/1a12a78e-90fe-48de-85e0-294e56fd232a)